### PR TITLE
MySQL 2 Hive operator

### PR DESCRIPTION
**Pull request by mistercrunch**
@askeys @artwr @jeremyclover 
New operator. The `HiveCliHook.load_file` method should be pretty useful too

### CODE REVIEW

**Comment by askeys** [comment]
@mistercrunch this looks very useful.  It would be good to make it clear in the docs that this stores the table as a textfile, and so typically you would add another hive operator to create a faster table.

**Comment by mistercrunch** [comment]
@askeys I took the time to improve the docs

I'm happy with how HiveCliHook.load_file should be useful on its own and gets reused for all the ".*ToHiveTransfer" operators.

**Comment by askeys** [comment]
Efficiency :+1:

File: airflow/hooks/hive_hooks.py [hunk 1]
```diff
@@ -60,6 +60,62 @@ def run_cli(self, hql, schema=None):
             if sp.returncode:
                 raise Exception(all_err)
 
+    def load_file(
+            self,
+            filepath,
+            table,
+            delimiter="'",
```
Comment (artwr) [inline airflow/hooks/hive_hooks.py]: Is the default delimiter really ' ? How about \t or ^A?

**Comment by mistercrunch** [comment]
Good catch, I meant it to be a comma assuming the adhoc csv use case for the hook. I'll replace by a comma now. The default for the operator is ^A (`chr(1)`)

**Comment by mistercrunch** [comment]
@artwr , I push a commit straight to master here:
https://github.com/mistercrunch/Airflow/commit/d01c4c8560f520c9c9755026cf88a7baff6300e3

**Pull request merged**